packages/terra-application/CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## Unreleased
 
-* Updated
+## 1.56.0 - (August 22, 2023)
+
+* Changed
   * Updated `ApplicationBase` documentaion.
```

Comment:
Do you think these could have been patch releases instead of minor?

Reply:
I think this makes sense to be a patch. Changed to a patch version in 8fa442acfd4bbc1dc09b9145de99849230f69288 & 2c1144007878f7ccb06762228228cb6ecd21e53b. The rest I think are fine as minor versions.